test/eunit/yaws_ls_tests.erl

```diff
@@ -0,0 +1,13 @@
+%% @doc `yaws_ls_tests' -- eunit tests for `yaws_ls'
+
```

Comment:
I would get rid of lines 1 and 2 here and replace them with `%% -*- coding: latin-1 -*-` (or whatever setting is appropriate).